src/math/p5.Vector.js

```diff
@@ -34,14 +34,14 @@ class Vector {
   // This is how it comes in with createVector()
   // This check if the first argument is a function
   constructor(...args) {
-    let dimensions = args.length; // TODO: make default 3 if no arguments
```

Comment:
With `createVector`, this would be e.g. 5 for a 3-dimensional vector. As it is, there are no uses of the `dimensions` property, which is likely why it went by unnoticed.

Reply:
Good catch!